src/context/NetworkContext.tsx

```diff
@@ -30,6 +32,7 @@ const LitentryNetworkTest: NetworkType = {
   ws: ['wss://staging.registrar.litentry.io'],
   isTestnet: true,
   color: '#006400',
+  ss58Format: 9999, // don't know yet
```

Comment:
It's `31`, you can find it [here](https://github.com/paritytech/substrate/blob/master/ss58-registry.json)